tests/yam/agama/boot_encrypt.pm

```diff
@@ -0,0 +1,19 @@
+## Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Boot agama encryption.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::agama_base;
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    wait_still_screen 5;
+    save_screenshot;
```

Comment:
you just need to borrow the original code that is executed here:
https://openqa.suse.de/tests/15533450#step/grub_test/1 and I don't see any needle and this file or the other one, could list all the relevant commands in the original code to compare?